### PR TITLE
Adding templates/concatenation

### PR DESCRIPTION
**Pull request by a-e-m**
The code is a little messy, but it works!

### CODE REVIEW

**Comment by trevorgrayson** [comment]
could you rebase? Sorry! :(

**Comment by a-e-m** [comment]
Sure, will do. Git seems to have gotten a bit confused

File: src/main/resources/movie.ftl [hunk 1]
```diff
@@ -3,6 +3,7 @@ id: !!str TMSId
 fallback: nope|rootId
 title: titles/title*
 description: descriptions/desc
+description_es: descriptions/desc[@lang='es']
```
Comment (trevorgrayson) [inline src/main/resources/movie.ftl]: i'm trying to abstract away the attribute tags.  Can we do this without "@"?  

for the other elements i've been looking for an element (xpath descriptions/desc/lang) and if that doesn't exist, i check for attrs (descriptions/desc/@lang)

i can't think of other formats with attrs

**Review by trevorgrayson — COMMENTED** [review]
could you rebase? sorry!

File: src/main/resources/movie.ftl (hunk 1 above)
Comment (a-e-m) [reply src/main/resources/movie.ftl]: Yeah I think we should do this without @, this is something I left in by accident when I was just trying things

**Pull request closed without merging**